packages/flutter/lib/src/widgets/dismissible.dart

```diff
@@ -62,6 +62,39 @@ enum DismissDirection {
 ///
 /// {@youtube 560 315 https://www.youtube.com/watch?v=iEMgjrfuc58}
 ///
+/// {@tool dartpad --template=stateful_widget_scaffold}
+///
+/// This sample shows how you can use the [Dismissible] widget to
+/// remove list items using swipe gestures.
+///
+/// ```dart
+///  List<int> items = List<int>.generate(100, (index) => index);
+///
+///  Widget build(BuildContext context) {
+///    return ListView.builder(
+///      itemCount: items.length,
+///      padding: const EdgeInsets.symmetric(vertical: 16),
+///      itemBuilder: (context, index) => Dismissible(
```

Comment:
Define the types in the arguments:
```suggestion
///      itemBuilder: (BuildContext context, int index) => Dismissible(
```